templates/emails/transactional/unsubscribe-newsletter.txt

```diff
@@ -1,12 +1,9 @@
+{% extends "base.txt.jinja2" %}
+
+{% block content %}
 You have been unsubscribed from SimpleLogin newsletter.
 
 We would love to know why do you want to unsubscribe from our communication.
 
 Please let us know by replying to this email.
```

Comment:
Nits: we can remove the "Please let us know by replying to this email." here

Reply:
Ooops. Removed